src/parser.rs

```diff
@@ -37,6 +37,44 @@ use crate::ParsedDaysOfWeek;
 use crate::PossibleLiterals;
 use crate::PossibleValue;
 
+/// Options to manipulate the parsing manner.
+#[derive(Debug, Copy, Clone)]
+pub struct ParseOptions {
+    /// Whether fallback to UTC when the timezone part is missing.
+    ///
+    /// Default to `false`.
+    pub fallback_to_utc_timezone: bool,
```

Comment:
I don't think this mode should even be supported. If users _really_ want this, they can with `TZ=UTC` or something. But assuming UTC is usually pretty unintuitive.

If you really want to keep this, please add some docs about this generally not being what end users are going to expect.

Reply:
This is about different flavor...

Supporter for use UTC as a default:

* https://crontab.guru/tips.html Tip 2
* http://yellerapp.com/posts/2015-01-12-the-worst-server-setup-you-can-make.html

I follow Snowflake's flavor and require the timezone to be specified as described in the FAQ.

I'll consider make this option an enum per the other comment.